### DISCUSSION TITLE
Send to multiple Teams channels based on notification?

### DISCUSSION THREAD

**Discussion by ssnyder** [excid3/noticed#290] [Q&A]
Just found this gem so I haven't finished looking into my question, but thought I would ask here to start....

I want to publish internal errors/issues to a developer channel on Teams, but informative notifications to stakeholders - is this possible? How would I set this up?

**Comment by excid3** [comment]
If you mean separate notifications for each, you would create two separate Notification classes and send them as needed.

Or, if you mean delivering a notification with the same delivery method We store the delivery methods as an array, so you should be able to use the same one twice with different options.

```ruby
class ErrorNotification < Noticed::Base
  deliver_by :microsoft_teams, url: :developers_channel_url
  deliver_by :microsoft_teams, url: :stakeholder_channel_url

  def developers_channel_url
     # returns url
  end

  def stakeholder_channel_url
    # returns url
  end
end
```
